Sources/TuistLoader/Models+ManifestMappers/Config+ManifestMapper.swift

```diff
@@ -19,8 +19,14 @@ extension TuistCore.Config {
         if let manifestCloud = manifest.cloud {
             cloud = try TuistCore.Cloud.from(manifest: manifestCloud)
         }
+
+        var cache: TuistCore.Cache?
+        if let manifestCache = manifest.cache {
+            cache = TuistCore.Cache.from(manifest: manifestCache)
+        }
         return TuistCore.Config(compatibleXcodeVersions: compatibleXcodeVersions,
                                 cloud: cloud,
+                                cache: cache ?? .default,
```

Comment:
I am not sure, if providing a default cache in a mapper is a good place of doing so, thoughts? 🤔

Reply:
I suggested this in another comment but yeah, I think it should be done in the `CacheWarmService`.

Reply:
Will do 👍